### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
Added Login Signup pages .

### CODE REVIEW

**Comment by Mohasin-Haque** [comment]
recording


<details><summary>Browser metadata</summary>





```
Path:      /signup
Browser:   Chrome 100.0.4896.88 on Windows 10
Viewport:  1366 x 617 @1x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-7--fast-cart-react.netlify.app%2Fsignup&os=Windows&os_version=10&browser=Chrome&browser_version=100.0&resolution=1366x617&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-7--fast-cart-react.netlify.app/signup) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=625c1ff462c59e000abcb158&commentID=625c2493d3dd48672ac00877&resolution=resolved)

File: src/pages/landing-page/LandingPage.jsx [hunk 1]
```diff
@@ -20,7 +20,6 @@ const LandingPage = () => {
             console.error(error, "Happy Diwali, API phat gyi!!");
         }
```
Comment (amirnathani9) [inline src/pages/landing-page/LandingPage.jsx:21]: dont write this type of messages in your main projects

File: src/pages/landing-page/LandingPage.jsx [hunk 2]
```diff
@@ -39,17 +38,29 @@ const LandingPage = () => {
...
+                            <Link className="btn btn-primary" to="/products">Silk Special</Link>
+                            <Link className="btn btn-secondary" to="/products">Wedding Special</Link>
                         </div>
                     </div>
                 </section>
             </main>
+            <section className="featured-section">
+            <h3 className="heading heading-avatar heading-card heading-ecom"> </h3>
+                <div className="featured-product">
+                    <img className="responsive" src={Ban2} alt="error" />
```
Comment (amirnathani9) [inline src/pages/landing-page/LandingPage.jsx:50]: didnt understand Ban2, you can write in discription

File: src/pages/landing-page/LandingPage.jsx [hunk 3]
```diff
@@ -39,17 +38,29 @@ const LandingPage = () => {
...
-                    <FeaturedCard />
-                    
+                {categories.map((product) => ( <FeaturedCard key={product._id} product={product} /> )
+                )}     
+                </div>
+            </section>
+            <section className="featured-section">
+            <h3 className="heading heading-avatar heading-card heading-ecom"> </h3>
+                <div className="featured-product">
+                    <img className="responsive" src={Ban3} alt="error" />
```
Comment (amirnathani9) [inline src/pages/landing-page/LandingPage.jsx:63]: same upper changes

**Review by amirnathani9 — APPROVED** [review]
some neat peaks . good to merge
- code structure is good

File: src/pages/landing-page/LandingPage.jsx [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { Navbar, CategoryCard, FeaturedCard } from "../../components/index";
 import { Link } from "react-router-dom";
 import axios from "axios";
 import { useState, useEffect } from "react";
+import {  Ban2, Ban3 } from "../../assets";
```
Comment (yogesh-) [inline src/pages/landing-page/LandingPage.jsx:6]: Give proper names to variables.

**Review by yogesh- — APPROVED** [review]
Suggested minor changes
- UI is nice
- made smaller components
- code is readable

File: src/pages/landing-page/LandingPage.jsx (hunk 2 above)
Comment (Mohasin-Haque) [reply src/pages/landing-page/LandingPage.jsx:50]: Its banner . I saw most of the sites using this convention for banners so did it.

**Pull request merged**